packages/helm-charts/common/templates/_helpers.tpl

```diff
@@ -28,3 +82,98 @@
     mountPath: "/root/.celo/account"
     readOnly: true
 {{- end -}}
+
+{{- define "common.full-node-container" -}}
+- name: geth
+  image: {{ .Values.geth.image.repository }}:{{ .Values.geth.image.tag }}
+  imagePullPolicy: {{ .Values.geth.image.imagePullPolicy }}
+  command:
+  - /bin/sh
+  - -c
+  args:
+  - |
+    set -euo pipefail
+    RID=$(echo $REPLICA_NAME | grep -Eo '[0-9]+$')
+    public_ip=$(echo "$PUBLIC_IPS" | awk -v RID=$(expr "$RID" + "1") '{split($0,a,","); print a[RID]}')
+    NAT_FLAG="--nat=extip:${public_ip}"
+
+    ADDITIONAL_FLAGS='{{ .geth_flags | default "" }}'
+    {{ if .proxy }}
+    VALIDATOR_HEX_ADDRESS=$(cat /root/.celo/validator_address)
+    ADDITIONAL_FLAGS="--proxy.proxiedvalidatoraddress $VALIDATOR_HEX_ADDRESS --proxy.proxy --proxy.internalendpoint :30503 $PROXY_ALLOW_PRIVATE_IP_FLAG"
+    {{ end }}
+    {{- if .unlock | default false }}
+    ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --unlock=${ACCOUNT_ADDRESS} --password /root/.celo/account/accountSecret --allow-insecure-unlock"
+    {{- end }}
+    {{- if .expose }}
+    RPC_APIS="{{ .rpc_apis | default "eth,net,web3,debug,txpool" }}"
+    ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --rpc --rpcaddr 0.0.0.0 --rpcapi=${RPC_APIS} --rpccorsdomain='*' --rpcvhosts=* --ws --wsaddr 0.0.0.0 --wsorigins=* --wsapi=${RPC_APIS}"
+    {{- end }}
+    {{- if .ping_ip_from_packet | default false }}
+    ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --ping-ip-from-packet"
+    {{- end }}
+    {{- if .in_memory_discovery_table_flag | default false }}
+    ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --use-in-memory-discovery-table"
+    {{- end }}
+    {{- if .proxy_allow_private_ip_flag | default false }}
+    ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --proxy.allowprivateip"
+    {{- end }}
+  
+    geth \
+      --bootnodes=$(cat /root/.celo/bootnodeEnode) \
+      --light.serve 90 \
+      --light.maxpeers 1000 \
+      --maxpeers 1100 \
```

Comment:
Could we make `light.serve`, `light.maxpeers`, and `maxpeers` configurable? I imagine down the road these 3 might need to be changed depending on how many peers we want for the full nodes